src/app-layout/__tests__/utils.tsx

```diff
@@ -284,3 +284,32 @@ export const singleDrawerPublic: Array<AppLayoutProps.Drawer> = [
     },
   },
 ];
+
+export const multipleDrawers: Array<AppLayoutProps.Drawer> = [
```

Comment:
Is there a reason you need to add this instead of using `singleDrawerPublic`?

Reply:
I added it to simulate a real use-case with 2 drawers, when one of them is a help panel. But for the test it doesn't really matter, you are right :)